test/unit/carriers/shipwire_test.rb

```diff
@@ -126,7 +126,7 @@ def test_validate_credentials_with_invalid_credentials
   end
 
   def test_rate_request_includes_address_name_if_provided
-    name = CGI.escape("<Name><Full>Bob Bobsen</Full></Name>")
+    name = CGI.escape("<Full>Bob Bobsen</Full>")
```

Comment:
The `<Name>` tag is still there but with a bunch of whitespace in between. Given that it just wants to make sure the name is included I though this part would be enough to verify that.